corehq/ex-submodules/casexml/apps/case/xform.py

```diff
@@ -152,7 +152,7 @@ def _get_all_dirtiness_flags_from_cases(domain, case_db, touched_cases):
     # process the temporary dirtiness flags first so that any hints for real dirtiness get overridden
     if domain:
         domain_obj = Domain.get_by_name(domain)
-        if domain_obj and domain_obj.use_livequery:
+        if domain_obj and domain_obj.use_livequery or toggles.LIVEQUERY_SYNC.enabled(domain):
```

Comment:
nit: to make the reader not have to remember if `and` or `or` binds more tightly and/or if it's determined by the order, parenthesis would be nice

Reply:
In fact

```
>>> False and False or True
True
```
implies that it's interpreted as `(A and B) or C` and not `A and (B or C)`, so I think the parens are actually necessarily in addition to making it more readable.